src/Advection/flat_advective_fluxes.jl

```diff
@@ -19,6 +19,10 @@ for SchemeType in [:CenteredScheme, :UpwindScheme]
     end
 end
 
+const XFlatGrid = AG{<:Any, Flat}
+const YFlatGrid = AG{<:Any, <:Any, Flat}
+const ZFlatGrid = AG{<:Any, <:Any, <:Any, Flat}
+
```

Comment:
using Oceananigans.Operators: XFlatGrid, YFlatGrid, ZFlatGrid

Reply:
done

Reply:
ah ok you have already done it :sweat_smile: